globus_automate_client/cli/callbacks.py

```diff
@@ -224,3 +221,44 @@ def input_validator_callback(body: str) -> str:
         raise typer.BadParameter("Expected file, received directory")
 
     return body
+
+
+def flow_input_validator(body: str) -> str:
```

Comment:
Nice catch that plain JSON can get loaded by the YAML loader. Does this mean we can remove the other more specific json/yaml loaders to do something more generic as is here?

Reply:
I think it's okay to remove both of the yaml and json validator callbacks and my first instinct is to remove them, but I wasn't sure how you all handled no longer used code. Sometimes people preserve it for future use, but I'm not sure that's necessary here

Reply:
I feel similarly. If we need to we can return to git history.